app/src/main/java/cs5150athletetracking/com/athletetracking/Uploader.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Uploader implements Runnable {
+	// URL should be updated. We are currently using a test script "test_server.php"
+	private static final String URL = "http://54.165.208.160/php/test_server.php";
+	private static final String TAG_SUCCESS = "success";
+	private String timestamp;
+	private String username;
+	private String bibnumber;
+	private String raceid;
+	private String latitude;
+	private String longitude;
+	private String altitude;
+
+	public Uploader(String user, int bib, int race, double lat, double lon, double alt) {
```

Comment:
For instance, we either pass in the success (true or false) or the callback is an object with an onSuccess and onFailure interface.